galleries/views.py

```diff
@@ -11,6 +10,7 @@ def galleries(request):
         'tags':tags
```

Comment:
If you are using variable only in one place and it is not a huge object it is easier to just embed it in the place of usage

```
return render(request, 'galleries.html', {'galleries': galleries, 'tags': tags})
```